communication/slack-config/channels.yaml

```diff
@@ -342,6 +342,7 @@ channels:
   - name: wg-policy
   - name: wg-resource-mgmt
   - name: wg-security-audit
+  - name: wg-service-binding
```

Comment:
+1 with the channel, but to use a `wg-` prefix, you need to be an "official" working group in the [sig list](https://github.com/kubernetes/community/blob/master/sig-list.md)

Reply:
If you don't have the intention to create a WG, remove the prefix would be good. But if you do have that intention, either you can wait for it formed, or create without prefix now and rename it later.

Reply:
Sure, I will go ahead by removing the prefix for now.